digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/AddressLinkBuilder.scala

```diff
@@ -14,13 +14,6 @@ trait AddressLinkBuilder {
   val formatter: DateTimeFormatter = DateTimeFormat.forPattern("dd.MM.yyyy")
   val dateTimeFormatter: DateTimeFormatter = ISODateTimeFormat.dateOptionalTimeParser()
 
-  lazy val municipalityMapping: Map[Long, Long] = if (PostGISDatabase.isWithinSession)
-    MunicipalityDAO.getMunicipalityMapping
-  else
-    PostGISDatabase.withDynSession {
-      MunicipalityDAO.getMunicipalityMapping
-    }
-
```

Comment:
Kuollutta koodia

Reply:
Pitäiskö vielä kommentoida tähän että on viiteen-käyttämä ely-koodi tuossa maintainerMappingissa, tms.

Reply:
Pitäis. :) 
Mä oon säätämässä VIITE-2956-related-housekeeping -branchissa kommentaareja, voi jättää täältä pois.

Reply:
Eikös olis parempi lisätä tyypitys, joko case class tai Type
, https://alvinalexander.com/scala/scala-type-examples-type-aliases-members/